test/verify/check-storage-lvm2

```diff
@@ -900,5 +806,121 @@ class TestStorageLvm2(storagelib.StorageCase):
         b.wait_in_text(self.card_desc("LVM2 logical volume", "Physical volumes"), bn(disk))
 
 
+class TestStorageLvm2Destructive(storagelib.StorageCase):
+
+    def can_do_layouts(self):
+        return self.storaged_version >= [2, 10]
+
+    def skip_if_no_layouts(self):
+        if not self.can_do_layouts():
+            raise unittest.SkipTest("raid layouts not supported")
+
+    # This is only "destuctive" because of
```

Comment:
destr**u**ctive

Most typos in comments don't matter, but that may be one which we want to grep for

Reply:
Yep.